Makefile

```diff
@@ -184,11 +184,23 @@ test-all: test-build test/gc/node_modules/weak/build/Release/weakref.node
 test-all-valgrind: test-build
 	$(PYTHON) tools/test.py --mode=debug,release --valgrind
 
+CI_NATIVE_SUITES := addons
+CI_JS_SUITES := doctool known_issues message parallel pseudo-tty sequential
+
+test-ci-native: | test/addons/.buildstamp
```

Comment:
`build-addons` triggers compilation of the node executable. This cannot happen because this target will run on the test machine, after the cross-compiled binary is unpacked, and make will recompile everything.